### PR TITLE
Vs 2817 fix

### PR DESCRIPTION
**Pull request by vetrabmercedes**
https://cabidev.atlassian.net/browse/VS-2817

### CODE REVIEW

File: react/features/app/components/AbstractApp.js [hunk 1]
```diff
@@ -174,6 +178,8 @@ export class AbstractApp extends BaseApp<Props, *> {
      * @returns {void}
      */
     _connectToXmppServer() {
+        console.log('_connectToXmppServer');
```
Comment (bruderbarna) [inline react/features/app/components/AbstractApp.js]: Please remove unnecessary logging

File: react/features/app/components/AbstractApp.js [hunk 2]
```diff
@@ -223,6 +230,7 @@ export class AbstractApp extends BaseApp<Props, *> {
                     = JSON.parse(dataJsonString);
 
                 this.props.url.room = roomName;
+                console.log('------- VIDEOCONF_JOIN');
```
Comment (bruderbarna) [inline react/features/app/components/AbstractApp.js]: Please remove unnecessary logging

File: react/features/base/participants/middleware.js [hunk 3]
```diff
@@ -155,9 +159,18 @@ StateListenerRegistry.register(
...
-                && dispatch(participantLeft(p.id, p.conference));
+            ) {
+                const action = participantLeft(p.id, p.conference);
+
+                logger.info('would dispatch participant left because user is supposedly not in conference: ', action);
+                logger.info('user\'s conference', p.conference);
+                logger.info('current conference', conference);
+
+                // dispatch(action);
+            }
```
Comment (bruderbarna) [inline react/features/base/participants/middleware.js:163]: Please remove this whole state listener registration. It does not do anything anymore.

File: react/features/base/redux/middleware.js [hunk 4]
```diff
@@ -46,3 +49,15 @@ MiddlewareRegistry.register(store => next => action => {
...
+    logger.info('dispatching action: ', action);
+    const res = next(action);
+
+    logger.info('new features/base/conference state: ',
+        store.getState()['features/base/conference']);
+    logger.info('new features/base/participants state: ',
+        store.getState()['features/base/participants']);
+
+    return res;
+});
```
Comment (bruderbarna) [inline react/features/base/redux/middleware.js]: Please undo these changes, they were just for debugging

**Review by bruderbarna — APPROVED** [review]

File: react/features/base/participants/middleware.js (hunk 3 above)
Comment (vetrabmercedes) [reply react/features/base/participants/middleware.js:163]: Are we sure about that? We don't know exactly when this case will be true.

**Comment by vetrabmercedes** [comment]
https://jenkins.int.sonrisa.hu/job/jitsi-meet/28

**Review by molnartimi — APPROVED** [review]

**Review by kecskesk — APPROVED** [review]
